build.gradle

```diff
@@ -121,8 +121,10 @@ allprojects {
 		}
 
 		license {
-			header rootProject.file('HEADER')
+			header = rootProject.file('HEADER')
```

Comment:
lmao unneeded change

Reply:
i actually nuked thwe whole thing a couple of times
ended up with `=`